### ISSUE TITLE
Remove duplicates

### ISSUE THREAD

**Issue by ericclemmons** [geowarin/friendly-errors-webpack-plugin#30]
Some modules depend on the same thing, so the user sees results like:

```
These dependencies were not found in node_modules:

* js-cookie in ./path/to/something/different...
* color in ./path/to/something/different...
* mobx-react in ./path/to/something/different...
* react-component-queries in ./path/to/something/different...
* styled-components in ./path/to/something/different...
* styled-components in ./path/to/something/different...
* styled-components in ./path/to/something/different...
* styled-components in ./path/to/something/different...
* styled-components in ./path/to/something/different...
* styled-components in ./path/to/something/different...
* styled-components in ./path/to/something/different...
* styled-components in ./path/to/something/different...
* color in ./path/to/something/different...
* styled-components in ./path/to/something/different...
* styled-components in ./path/to/something/different...
* mobx in ./path/to/something/different...
* styled-components in ./path/to/something/different...
* styled-components in ./path/to/something/different...
* color in ./path/to/something/different...
* styled-components in ./path/to/something/different...
* styled-components in ./path/to/something/different...
* color in ./path/to/something/different...
* styled-components in ./path/to/something/different...
* styled-components in ./path/to/something/different...

Did you forget to run npm install --save for them?
```

A couple recommendations:

- [ ] `.filter` the errors to only show one per module (that's enough).
- [ ] Recommend `npm install --save ${modules.join(" "}`.
- [ ] For bonus points, check for `yarn.lock` in `process.cwd()` and recommend `yarn add ${modules.join(" "}` instead.

**Comment by geowarin** [comment]
Good idea !
I don't have time to work on this right now but it should be quite easy to do.

**Comment by geowarin** [comment]
Released in v1.5.0
